examples/helloworld/dbapi.py

```diff
@@ -23,3 +22,7 @@
 # in case run following code, it will not stop due to streaming query
 # for row in cusor:
 #     pprint(row)
+
+cusor = conn.execute("select 1=2")
```

Comment:
what does `select 1=2` mean? It's not so common

Reply:
it is logical expression , return `False`, check the `bool` type

Reply:
or maybe you want to create a bool, then 
`select 1<>2` will be better